ui/src/App.js

```diff
@@ -39,6 +40,9 @@ function App() {
             <Router history={history}>
               <Switch>
                 <Route exact path="/">
+                  <LandingPage />
+                </Route>
+                <Route path="/order">
```

Comment:
We will be using navigation service from now on. So this will be `path={getOrderPath()}`. Same for all other such places.